### PR TITLE
12.0 mig base tecnical user

### PR DESCRIPTION
**Pull request by GoodERPJeff**

### CODE REVIEW

File: base_technical_user/README.rst [hunk 1]
```diff
@@ -0,0 +1,53 @@
+.. image:: https://img.shields.io/badge/license-AGPL--3-blue.png
```
Comment (elicoidal) [inline base_technical_user/README.rst:1]: see https://github.com/OCA/server-tools/pull/1392#pullrequestreview-164502064 : many remarks might apply here

File: base_technical_user/README.rst [hunk 2]
```diff
@@ -0,0 +1,53 @@
...
+===================
+Base Technical User
+===================
+
+This module extends the functionality of company management.
+It allows you to bind a technical user on the company in order to use it in
+batch processes.
+
+The technical user must
+- be inactive to avoid login
```
Comment (elicoidal) [inline base_technical_user/README.rst:14]: add blank line

File: base_technical_user/README.rst [hunk 3]
```diff
@@ -0,0 +1,53 @@
...
+
+===================
+Base Technical User
+===================
+
+This module extends the functionality of company management.
+It allows you to bind a technical user on the company in order to use it in
+batch processes.
+
+The technical user must
```
Comment (elicoidal) [inline base_technical_user/README.rst:13]: add ":"

File: base_technical_user/README.rst [hunk 4]
```diff
@@ -0,0 +1,53 @@
...
+   :alt: Odoo Community Association
+   :target: https://odoo-community.org
+
+This module is maintained by the OCA.
+
+OCA, or the Odoo Community Association, is a nonprofit organization whose
+mission is to support the collaborative development of Odoo features and
+promote its widespread use.
+
+To contribute to this module, please visit http://odoo-community.org.
```
Comment (elicoidal) [inline base_technical_user/README.rst:53]: https

File: base_technical_user/__manifest__.py [hunk 5]
```diff
@@ -0,0 +1,19 @@
...
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl).
+{
+    'name': "Base Technical User",
+    'summary': """
+        Add a technical user parameter on the company """,
+    'author': 'ACSONE SA/NV, Odoo Community Association (OCA)',
+    'website': "http://acsone.eu",
+    'category': 'Hidden/Dependency',
+    'version': '12.0.1.0.0',
+    'license': 'AGPL-3',
```
Comment (elicoidal) [inline base_technical_user/__manifest__.py:11]: add cover image with screenshot

File: base_technical_user/__manifest__.py [hunk 6]
```diff
@@ -0,0 +1,19 @@
+# Copyright 2017 ACSONE SA/NV (<http://acsone.eu>)
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl).
+{
+    'name': "Base Technical User",
+    'summary': """
+        Add a technical user parameter on the company """,
+    'author': 'ACSONE SA/NV, Odoo Community Association (OCA)',
+    'website': "http://acsone.eu",
```
Comment (elicoidal) [inline base_technical_user/__manifest__.py:8]: https://github.com/OCA/server-tools/

File: base_technical_user/views/res_company_view.xml [hunk 7]
```diff
@@ -0,0 +1,20 @@
+<?xml version="1.0" encoding="utf-8"?>
+<odoo>
```
Comment (elicoidal) [inline base_technical_user/views/res_company_view.xml:2]: add copyright

**Review by elicoidal — CHANGES_REQUESTED** [review]
LGTM
some details

**Comment by pedrobaeza** [comment]
I'm not sure this module is still needed in v12, as now `admin` is a normal user.

**Comment by GoodERPJeff** [comment]
the module is trying to manage user for cron job in each company. so I think odoo admin user will not cover this scenario.

**Comment by GoodERPJeff** [comment]
the module is trying to manage user for cron job in each company. so I think odoo admin user will not cover this scenario.

**Comment by GoodERPJeff** [comment]
the module is trying to manage user for cron job in each company. so I think odoo admin user will not cover this scenario.

**Comment by GoodERPJeff** [comment]
the module is trying to manage user for cron job in each company. so I think odoo admin user will not cover this scenario.

**Comment by pedrobaeza** [comment]
OK, I didn't know the details of the module.

**Comment by florian-dacosta** [comment]
Is not the readme supposed to be generated automatically now? Or is it not mandatory?
@GoodERPJeff Will you adjust your PR with @elicoidal comments?

**Comment by pedrobaeza** [comment]
Superseeded by #1560

**Pull request closed without merging**